### DISCUSSION TITLE
React Oxide - built with this project

### DISCUSSION THREAD

**Discussion by ghost** [suren-atoyan/monaco-react#291] [General]
Hello,

I integrated this project with tabs and file management using webDAV.

https://github.com/bootrino/reactoxide

Thanks!

**Comment by suren-atoyan** [comment]
Hello @bootrino, look good 🙂 

As you use different files/languages you may be interested in [multi-model editor](https://github.com/suren-atoyan/monaco-react#multi-model-editor). Also, I would suggest using any UI framework; for example, [Material-UI](https://mui.com/) or [Chakra-UI](https://chakra-ui.com/) (or any other). You may not like their default look, but all of them are configurable, so you can change the look to suit your needs. But having a well-designed UI framework as the base of your application will make it look more professional - with better UX and accessibility.
